docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -103,69 +103,73 @@ Along with the HTTP error code, our API can also return more detailed error code
-| 50006  | Cannot send an empty message                                                                                    |
-| 50007  | Cannot send messages to this user                                                                               |
-| 50008  | Cannot send messages in a voice channel                                                                         |
-| 50009  | Channel verification level is too high                                                                          |
-| 50010  | OAuth2 application does not have a bot                                                                          |
-| 50011  | OAuth2 application limit reached                                                                                |
-| 50012  | Invalid OAuth state                                                                                             |
-| 50013  | Missing permissions                                                                                             |
-| 50014  | Invalid authentication token                                                                                    |
-| 50015  | Note is too long                                                                                                |
-| 50016  | Provided too few or too many messages to delete. Must provide at least 2 and fewer than 100 messages to delete. |
-| 50019  | A message can only be pinned to the channel it was sent in                                                      |
-| 50020  | Invite code is either invalid or taken.                                                                         |
-| 50021  | Cannot execute action on a system message                                                                       |
-| 50025  | Invalid OAuth2 access token                                                                                     |
-| 50034  | A message provided was too old to bulk delete                                                                   |
-| 50035  | Invalid Form Body                                                                                               |
-| 50036  | An invite was accepted to a guild the application's bot is not in                                               |
-| 50041  | Invalid API version                                                                                             |
-| 90001  | Reaction blocked                                                                                                |
-| 130000 | Resource overloaded                                                                                             |
+| ------ | -------------------------------------------------------------------------------------------------------------------------------- |
+| 0      | General error (such as a malformed request body, amongst other things).                                                           |
```

Comment:
It doesn't quite make sense to use full stops here as these aren't complete sentences. These don't need to be changed